senza/cli.py

```diff
@@ -952,9 +952,13 @@ def status(stack_ref, region, output, w, watch):
                         except:
                             http_status = 'ERROR'
                     else:
-                        try:
-                            answers = dns.resolver.query(name, 'CNAME')
-                        except:
+                        for record_type in ["CNAME", "A", "AAAA"]:
```

Comment:
but this will still not work as the `answer.target.to_text()` will not contain the stack version (only an IP..)